src/components/Modal/Modal.tsx

```diff
@@ -335,9 +336,15 @@ class Modal extends React.Component<CombinedProps, State> {
   private handleIFrameLoad = (evt: React.SyntheticEvent<HTMLIFrameElement>) => {
     const iframe = evt.target as HTMLIFrameElement;
     if (iframe && iframe.contentWindow) {
-      this.setState({
-        iframeHeight: iframe.contentWindow.document.body.scrollHeight,
-      });
+      try {
+        this.setState({
+          iframeHeight: iframe.contentWindow.document.body.scrollHeight,
+        });
+      } catch (err) {
```

Comment:
Should this be prefixed with an `_` since we don't use the `err` var?

```suggestion
      } catch (_err) {
```

Reply:
We don't anywhere else and it doesn't complain because it's required.

Reply:
Because typescript supports [optional catch binding](https://mariusschulz.com/blog/optional-catch-binding-in-typescript) we don't need this variable at all:


```suggestion
      } catch {
```